### PR TITLE
Adjustments

### PR DESCRIPTION
**Pull request by dlowrey**
Cron jobs implemented and other small adjustments made.

### CODE REVIEW

File: src/cron-jobs/crons.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const cron = require('node-cron');
+const db = require('./db.js');
+const mailer = require('../mailer.js');
+
+/**
+* markUncompleted: search for and mark uncompleted appointments at 9pm every
```
Comment (NilsG-S) [inline src/cron-jobs/crons.js]: I think the specification says this needs to be 8:00pm. 9:00pm is for reports.

File: src/mailer.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ let transporter = mailer.createTransport({
     service: 'gmail',
     auth: {
         user: 'fakehealthcareforyou@gmail.com',
-        pass: ''
+        pass: 'se1teamnull'
```
Comment (NilsG-S) [inline src/mailer.js]: You put the password in here. I'm not entirely certain that it's possible to take it out anymore...

**Review by NilsG-S — CHANGES_REQUESTED** [review]
Just a few things. We should also consider removing the appointments from the `dummy-data.sql` script since they're at the wrong time and the front-end is working

File: src/mailer.js (hunk 2 above)
Comment (dlowrey) [reply src/mailer.js]: welp time to change the password

**Review by NilsG-S — APPROVED** [review]

**Pull request merged**